tasks/omnibus.py

```diff
@@ -379,3 +386,46 @@ def manifest(
         omnibus_s3_cache=False,
         log_level=log_level,
     )
+
+
+@task
+def rpath_edit(ctx, install_path, target_rpath_dd_folder, platform="linux"):
+    # Since bash and sh don't support **/* we are using glob
+    # to get all datadog files inside of one `file --mime-type` command
+    datadog_files = glob.glob(f"{install_path}/**/*", recursive=True)
+    with open("/tmp/ddargs", "w") as f:
+        f.write(" ".join(datadog_files))
+    files = ctx.run("cat /tmp/ddargs | xargs file --mime-type", hide=True).stdout
```

Comment:
Would this work without having to create temporary file here?

```suggestion
    files = ctx.run(f"echo {" ".join(datadog_files)} | xargs file --mime-type", hide=True).stdout
```

Reply:
Nope because of the arguments size limit